### ISSUE TITLE
On 2.3.0-rc1 text is fuzzy

### ISSUE THREAD

**Issue by qbit** [fyne-io/fyne#3464]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [X] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

![image](https://user-images.githubusercontent.com/68368/206496767-e6dfd6a0-babc-424e-b6e0-c2d92d415d43.png)
On the right is 2.3.0-rc1, left is 2.2.3 (2.2.4 looks similar). Screenshot was taken with 96x96 DPI at 1.30 scale.

### How to reproduce

This is on NixOS unstable:
![image](https://user-images.githubusercontent.com/68368/206497850-0f0418ef-9502-4f7b-b460-8842ffca7774.png)

macOS Ventura doesn't seem as fuzzy.

### Example code

fyne_demo is enough to exhibit the issue.

### Fyne version

2.3.0-rc1

### Go compiler version

1.17.13, 1.19.3

### Operating system

Linux

### Operating system version

NixOS 22.11

**Comment by obsti8383** [comment]
I have a similar issue with 2.3.0 on Windows, some of the text is "fuzzy" while other text seems fine. Tested in two different Windows machines using different resolutions and Windows scaling setting (no difference). See the "You won't be able..." line in the following screenshot.

![fyne230_fuzzy_text](https://user-images.githubusercontent.com/16605975/209793581-0ac3e6e8-6476-413b-8742-258694aaeabd.png)

**Comment by alexballas** [comment]
![Screenshot from 2023-01-08 15-43-51](https://user-images.githubusercontent.com/11015965/211199488-ca512182-ebd5-4a4c-ac7a-722df3c7e1e3.png)
I confirm that the text is fuzzy in some places.
v2.3.0
Ubuntu 18.04
Go 1.19.4

**Comment by Jacalz** [comment]
Can you still replicate this if you try `v2.3.5` and also on the `develop` branch? I think this might have been fixed already.

**Comment by andydotxyz** [comment]
Only check develop - it has a new text renderer :)

**Comment by Jacalz** [comment]
Anyone that wants to verify that it is fixed please?

**Comment by alexballas** [comment]
![Screenshot from 2023-10-27 01-40-36](https://github.com/fyne-io/fyne/assets/11015965/af29e351-31df-4970-8684-660c04cec5e1)
still is for me.
Please check the "License" word and "MIT"

**Fyne:** 2.4.1
**OS:** Ubuntu 18.04
**Go:** 1.21.3

**Comment by Jacalz** [comment]
Okay. Thanks for confirming

**Comment by andydotxyz** [comment]
It is an old issue - can it be confirmed to still be a problem? Might be the same as #4390

**Comment by obsti8383** [comment]
@andydotxyz Yes, at least it still was an issue with v2.6.2. And yes, it seems related to #4390 .
Screenshots with v2.6.2. attached.

<img width="642" height="332" alt="Image" src="https://github.com/user-attachments/assets/f369eedb-6ee4-400b-9efe-344ba9018c72" />
<img width="812" height="308" alt="Image" src="https://github.com/user-attachments/assets/5be5860c-cf1f-408f-819a-fb002d632e1d" />

**Comment by andydotxyz** [comment]
This was recently resolved and will be in v2.7.3
